rir/src/compiler/pir/instruction.h

```diff
@@ -937,6 +938,7 @@ class FLIE(Force, 2, Effects::Any()) {
 
 class FLI(CastType, 1, Effects::None()) {
   public:
+    unsigned cost() const override final { return 0; }
```

Comment:
cost of what?

Reply:
idk :) how expensive to execute, something... currently my idea was just to distinguish between instructions which will be scheduled in rir, and "virtual" ones.

but the idea was that maybe we can extend on it and use it for other optimizations. e.g. an add with environment could be more expensive than plain add. but I am not really sure. I would grow this as we need it...